Cargo.toml

```diff
@@ -65,6 +65,7 @@ zeroize = "1"
 abscissa_core = { version = "0.7", features = ["testing"] }
 byteorder = "1"
 rand = "0.7"
+parameterized = "1.0.1"
```

Comment:
I'd prefer not to add additional dependencies, even if they're `dev-dependencies`

Reply:
Ok, I removed it. Sacrificed a bit "neatness" of the test code, but I think it's fine this way too: https://github.com/iqlusioninc/tmkms/pull/644/commits/5cb01cf6fed914956d9f8d565c92609bcd897e20